metrics/isolator.hpp

```diff
@@ -0,0 +1,164 @@
+#ifndef __METRICS_ISOLATOR_MODULE_HPP__
+#define __METRICS_ISOLATOR_MODULE_HPP__
+
+#include <map>
+#include <string>
+#include <vector>
+
+#include <mesos/mesos.hpp>
+#include <mesos/module.hpp>
+
+#include <mesos/slave/isolator.hpp>
+
+#include <process/future.hpp>
+#include <process/owned.hpp>
+#include <process/process.hpp>
+
+#include <stout/duration.hpp>
+#include <stout/flags.hpp>
+#include <stout/hashset.hpp>
+#include <stout/json.hpp>
+#include <stout/option.hpp>
+#include <stout/try.hpp>
+
+#include <stout/flags/parse.hpp>
+
+namespace mesosphere {
+namespace dcos {
+namespace metrics {
+
+// TODO(greggomann): Remove this once we have an equivalent helper in stout.
+// See https://reviews.apache.org/r/68543/.
+template<typename T>
+Try<T> parse(const std::string& body)
+{
+  Try<JSON::Object> json = flags::parse<JSON::Object>(body);
```

Comment:
Same comment I made in the above review: `JSON::parse<JSON::Object>` would be better.